src/Components/WebAssembly/Authentication.Msal/src/Interop/AuthenticationService.ts

```diff
@@ -47,7 +47,8 @@ interface AuthorizeService {
 
 interface AuthorizeServiceConfiguration extends Msal.Configuration {
     defaultAccessTokenScopes: string[];
-    additionalScopesToConsent: string[]
+    additionalScopesToConsent: string[];
+    loginMode: LoginMode;
```

Comment:
Should this be a string now?
```suggestion
    loginMode: string;
```